eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -13,7 +13,7 @@
     <!-- TODO: Figure out what to do about the netcoreapp ref packages (these are probably being pulled
          in via implicit versions and net6 targeting projects (e.g. tests)
          https://github.com/dotnet/source-build/issues/3356 -->
```

Comment:
After investigating, this is caused from https://github.com/dotnet/sdk/issues/30272.